.github/scripts/check-migration.sh

```diff
@@ -11,12 +11,39 @@ main() {
     echo "Found changed storage schema files:"
     echo "$CHANGED_FILES"
 
-    # Check each changed storage schema file
+    # Check if changes are only visibility modifiers
+    local significant_changes=false
     while IFS= read -r file; do
         [ -z "$file" ] && continue
-        check_migration_file "$file"
+        if ! is_only_visibility_change "$1" "$2" "$file"; then
```

Comment:
Maybe you can deefine those variables below the `main()`, for better readability
```
local base_sha="$1"
local head_sha="$2"
...
if ! CHANGED_FILES=$(git diff --name-only "$base_sha" "$head_sha" ...
...
```
```suggestion
        if ! is_only_visibility_change "$base_sha" "$head_sha" "$file"; then
```